apps/table.html

```diff
@@ -96,16 +99,16 @@ <h1 class="h1">caMicroscope</h1>
 
 	<div class='p-2 bg-light' style="flex-grow: 1;">
 		<button type="button" class="btn btn-success float-right mb-2"
-			onclick="(()=>{location.reload();return false;})()">Reload</button>
+			onclick="(()=>{location.reload();return false;})()"> <i class="fas fa-sync-alt"></i> Reload</button>
```

Comment:
It would be better if you change location.reload() to initialize().

Reply:
Yeah you are right. I did not see that. 
I did not add location.reload(), it was there already.
I think it should be changed though.

Reply:
I'll change that in the next PR if no one else does it.